google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITStorageTest.java

```diff
@@ -3420,4 +3461,42 @@ public void testAutoContentTypeCreateFrom() throws IOException {
   public void testAutoContentTypeWriter() throws IOException {
     testAutoContentType("writer");
   }
+
+  @Test
+  public void testNotification() throws InterruptedException, ExecutionException {
+    String bucketName = RemoteStorageHelper.generateBucketName();
+    Bucket bucket = storage.create(BucketInfo.newBuilder(bucketName).setLocation("us").build());
+    Notification bucketNotification =
+        Notification.newBuilder(TOPIC)
+            .setCustomAttributes(CUSTOM_ATTRIBUTES)
+            .setPayloadFormat(PAYLOAD_FORMAT)
+            .build();
+    try {
+      assertThat(storage.listNotifications(bucketName)).isEmpty();
+      Notification notification = storage.addNotification(bucketName, bucketNotification);
+      assertThat(notification.getGeneratedId()).isNotNull();
+      assertThat(CUSTOM_ATTRIBUTES).isEqualTo(notification.getCustomAttributes());
+      assertThat(PAYLOAD_FORMAT.name()).isEqualTo(notification.getPayloadFormat().name());
+      assertThat(notification.getTopic().contains(TOPIC)).isTrue();
+
+      // Gets the notification with the specified id.
+      Notification actualNotification =
+          storage.getNotification(bucketName, notification.getGeneratedId());
+      assertEquals(notification, actualNotification);
+
+      // Retrieves the list of notifications associated with the bucket.
+      List<Notification> notifications = storage.listNotifications(bucketName);
+      assertThat(notifications.size()).isEqualTo(1);
+      assertThat(notifications.get(0).getGeneratedId())
+          .isEqualTo(actualNotification.getGeneratedId());
+
+      // Deletes the notification with the specified id.
+      boolean isDeleted = storage.deleteNotification(bucketName, notification.getGeneratedId());
+      assertThat(isDeleted).isTrue();
```

Comment:
The final nit from me:
      boolean isDeleted = storage.deleteNotification(bucketName, notification.getGeneratedId());
      assertThat(isDeleted).isTrue();
->
      assertThat(storage.deleteNotification(bucketName, notification.getGeneratedId())).isTrue();
      assertThat(storage.deleteNotification(bucketName, notification.getGeneratedId())).isFalse();

Reply:
done